plugin.video.embycon/resources/lib/downloadutils.py

```diff
@@ -336,7 +367,7 @@ def imageUrl(self, id, art_type, index, width, height, imageTag, server):
             artwork += '&MaxWidth=%s' % width
         if int(height) > 0:
             artwork += '&MaxHeight=%s' % height
-        return artwork
+        return artwork + "|verifypeer=false"
```

Comment:
same

Reply:
use the verify_cert setting to append the verifypeer=false is needed